### DISCUSSION TITLE
About the FEC restoration issue

### DISCUSSION THREAD

**Discussion by xiuer-1573** [alvr-org/ALVR#1189] [Q&A]
At present, when the data of each P-frame is less than 20 packets, the number of FEC packets is 1, so how to combat the situation that two packets are lost in a video frame? When more than one packet is lost, I found that the effect is good on ALVR, may I ask what is done on the client

**Comment by zmerp** [comment]
I'm not quite sure how the FEC code works. That was inherited from [polygraphene/ALVR](https://github.com/polygraphene/ALVR). I might put my hands on it soon (moving it to Rust) and so I might have a chance to study it.
